source/ed/ed_reader.cpp

```diff
@@ -63,12 +63,13 @@ void EdReader::fill(navitia::type::Data& data, const double min_non_connected_gr
     this->fill_lines(data, work);
     this->fill_line_groups(data, work);
     this->fill_routes(data, work);
-
     this->fill_validity_patterns(data, work);
-    this->fill_vehicle_journeys(data, work);
 
     //the comments are loaded before the stop time to reduce the memory foot print
     this->fill_comments(data, work);
+    this->fill_stop_times(data, work);
```

Comment:
It might be nice to add a comment to explain why do we load the stoptimes before the vj